config/config.go

```diff
@@ -662,6 +775,22 @@ var defaultConf = Config{
 		RecordPlanInSlowLog: logutil.DefaultRecordPlanInSlowLog,
 		EnableSlowLog:       *NewAtomicBool(logutil.DefaultTiDBEnableSlowLog),
 	},
+	Instance: Instance{
+		GeneralLog:                  false,
+		EnablePProfSQLCPU:           false,
+		DDLSlowOprThreshold:         DefDDLSlowOprThreshold,
+		ExpensiveQueryTimeThreshold: DefExpensiveQueryTimeThreshold,
+		EnableSlowLog:               *NewAtomicBool(logutil.DefaultTiDBEnableSlowLog),
+		SlowThreshold:               logutil.DefaultSlowThreshold,
+		QueryLogMaxLen:              logutil.DefaultQueryLogMaxLen,
```

Comment:
We plan to convert QueryLogMaxLen to a GLOBAL sysvar in https://github.com/pingcap/tidb/issues/33769

So I think we should remove it for now so we don't move it around twice.

(I checked with PM - it does sort of describe something that is usually instance scope, but we prefer GLOBAL here.)